doc/experimental/oam-to-kubernetes.md

```diff
@@ -1,11 +1,32 @@
 # OAM to Kubernetes Mappings
 
-### Version: v0.0.1-draft
+### Version: v0.0.2-draft
+This document explains how to generate Kuberenetes YAML files from OAM-related resources that are running in a Kubernetes cluster.
 
-## Verrazzano CLI export command
+## Download Verrazzano 1.6.11 CLI
 Verrazzano provides a CLI command that you can use to facilitate the migration of an OAM application to be managed as a collection of Kubernetes objects.
+If you don't have the 1.6.11 CLI (or later), you need to download it to export the OAM resources to Kubernetes manfests.
 
-The command vz export oam will output the YAML for each Kubernetes object that was generated as a result of deploying an OAM application. The generated YAML is sanitized so that it can be used to deploy the application. Fields such as creationTimestamp, resourceVersion, uid, and status are not included in the output.
+Download the CLI for Linux-AMD as follows:
+```
+curl -LO https://github.com/verrazzano/verrazzano/releases/download/v1.6.11/verrazzano-1.6.11-linux-amd64.tar.gz
```

Comment:
Should we make this command generic to any linux environment, or just use amd64 as an example?

Reply:
I just checked the VZ doc, and it just has amd64, so probably ok as is.